### ISSUE TITLE
Get-ADDBAccount : Parameter is not a hexadecimal string

### ISSUE THREAD

**Issue by MichaelGrafnetter** [MichaelGrafnetter/DSInternals#6]
Got this bug report through a different channel:

ntds.dit and SYSTEM is exported from a win2008R2 server using ntdsutil. Running the powershell module from a win2012R2.

Unblocked the module zip file before before installing.

PS C:> $key = Get-BootKey -SystemHivePath “C:\SYSTEM”
PS C:>
PS C:> $key
2bc5ae2c28662f04b23a33008c743be8
PS C:>
PS C:> Get-ADDBAccount -All -DBPath “C:\ntds.dit” -BootKey $key
Get-ADDBAccount : Parameter is not a hexadecimal string.
At line:1 char:1
- Get-ADDBAccount -All -DBPath “C:\ntds.dit” -BootKey $key
- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
- CategoryInfo : OpenError: (:) [Get-ADDBAccount], ArgumentException
- FullyQualifiedErrorId : DBContextError,DSInternals.PowerShell.Commands.GetADDBAccountCommand

**Comment by MichaelGrafnetter** [comment]
Probably a duplicate of bug #42.
